trieste/acquisition/optimizer.py

```diff
@@ -766,6 +740,70 @@ def get_bounds_of_box_relaxation_around_point(
     return spo.Bounds(space_with_fixed_discrete.lower, space_with_fixed_discrete.upper)
 
 
+def get_bounds_of_optimization(space: SearchSpace, starting_points: TensorType) -> List[spo.Bounds]:
```

Comment:
I would improve the docstring here if possible. 

I find it very difficult to understand what this does. Especially with the notion of "relaxed bounds".

I don't know if it is feasible at all but maybe even show what this does for a very simple example?

Reply:
I have improved the docstring and also added an example.